mnefun/_mnefun.py

```diff
@@ -1492,8 +1514,17 @@ def do_preprocessing_combined(p, subjects, run_indices):
         raw_orig = _raw_LRFCP(pre_list, p.proj_sfreq, None, bad_file,
                               p.n_jobs_fir, p.n_jobs_resample, projs, bad_file,
                               p.disp_files, method='fft',
-                              filter_length=p.filter_length, force_bads=False)
+                              filter_length=p.filter_length, force_bads=False,
+                              l_trans=p.hp_trans, h_trans=p.lp_trans)
+
+        # Apply any user-supplied extra projectors
+        if p.proj_extra is not None:
+            if p.disp_files:
+                print('    Adding extra projectors from "%s".' % p.proj_extra)
+            extra_proj = op.join(pca_dir, p.proj_extra)
+            projs = read_proj(extra_proj)
 
```

Comment:
We have a paradigm where users respond verbally at the end of each trial. We added a projector to get rid of the large artifact, as it bled into the baseline period a bit due to the acquisition high-pass filter.